### ISSUE TITLE
Showcase page

### ISSUE THREAD

**Issue by alfonsogarciacaro** [dbrattli/Fable.Reaction#1]
Great project with a nice README and a samples! :clap: :clap: Would it be possible to add a simple web using GH pages with the live samples (as in the GIFs you tweet)? That would be very helpful for users to get a quick grasp of Fable.Reaction benefits.

Also, please don't forget to add a link to your project in [awesome-fable](https://github.com/kunjee17/awesome-fable) so it's easier for other people to find it :)

**Comment by dbrattli** [comment]
Thanks @alfonsogarciacaro . Reaction started as a playground project to see if it would be possible to join the Elm-architecture with observables, and things are getting quite interesting so it looks like its turning into a real project. Still a bit confused about naming, if the project should be called Fable.Reaction or Elmish.Reaction and if the nuget should perhaps be called Fable.Elmish.Reaction. Decided to go for Elmish event-loop instead of rolling my own just a day ago, so the dust haven't settled yet. But it's a good idea to add a GH-page to show the demos (live) and better explain how things work.

**Comment by alfonsogarciacaro** [comment]
Indeed, this is a very promising projects. I've had issues myself when reacting to event streams in Elmish projects, so it would be awesome if you could combine both projects! About the name, Fable.Elmish changed it to Elmish when it started to be applied in other platforms (Xamarin, WPF). If you want to fuse it with Elmish and it can potentially be applied to other platform, `Elmish.Reaction` is a good candidate (as you say, the nuget can keep the Fable prefix) :+1:

**Comment by dbrattli** [comment]
There's actually been some progress to this issue 😄 I've been struggling with finding the right tooling and hosting for the docs, but after working on RxPY for a few months I've decided to move the docs to [Sphinx](http://www.sphinx-doc.org/en/master/) and [Read The Docs](https://readthedocs.org/), which also makes it easier to add custom Css and Js. Still some work left to make things look good, but consider this a first POC: 

https://fablereaction.readthedocs.io/en/latest/fable.elmish/demo.html
